modules/apps/app-builder/app-builder-test/src/testFunctional/macros/FormViewBuilder.macro

```diff
@@ -368,6 +368,13 @@ definition {
 			locator1 = "FormViewBuilder#FIELD_ELLIPSIS_ACTION");
 	}
 
+	macro editColor {
+		Type(
+			key_fieldName = "${fieldName}",
```

Comment:
@leilanyaragao, let's rename:

- `fieldName` to `fieldLabel`
- `key_fieldName` to `key_fieldLabel`